service-print/src/main/java/org/oskari/print/PDF.java

```diff
@@ -735,22 +735,22 @@ private static void draw(PDPageContentStream stream, Geometry g, PDPrintStyle st
         }
     }
 
-    private static void drawLabel(PDPageContentStream stream, Geometry g, String label) throws IOException {
-        Coordinate c;
+    private static void drawLabel(PDPageContentStream stream, PDPrintStyle.LabelAlign align, Geometry g, String label) throws IOException {
```

Comment:
Having two functions with the same name but similiar parameters in mixed order is confusing:
`void drawLabel(PDPageContentStream stream, PDPrintStyle.LabelAlign align, Geometry g, String label)`
`void drawLabel(PDPageContentStream stream, Coordinate c, PDPrintStyle.LabelAlign align, String label)`
Either:
`void drawLabel(PDPageContentStream stream, PDPrintStyle.LabelAlign align, Geometry g, String label)`
`void drawLabel(PDPageContentStream stream, PDPrintStyle.LabelAlign align, Coordinate c, String label)`
Or:
`void drawLabel(PDPageContentStream stream, Geometry g, PDPrintStyle.LabelAlign align, String label)`
`void drawLabel(PDPageContentStream stream, Coordinate c, PDPrintStyle.LabelAlign align, String label)`
Or rename either one to `drawLabelAt` or something like that.